packages/rln/src/rln_light.ts

```diff
@@ -0,0 +1,235 @@
+import { hmac } from "@noble/hashes/hmac";
+import { sha256 } from "@noble/hashes/sha256";
+import { Logger } from "@waku/utils";
+import { ethers } from "ethers";
+
+import { LINEA_CONTRACT } from "./contract/constants.js";
+import { RLNLightContract } from "./contract/rln_light_contract.js";
+import { IdentityCredential } from "./identity.js";
+import { Keystore } from "./keystore/index.js";
+import type {
+  DecryptedCredentials,
+  EncryptedCredentials
+} from "./keystore/index.js";
+import { KeystoreEntity, Password } from "./keystore/types.js";
+import {
+  buildBigIntFromUint8Array,
+  extractMetaMaskSigner
+} from "./utils/index.js";
+
+const log = new Logger("waku:rln");
+
+/**
+ * Create an instance of RLN
+ * @returns RLNInstance
+ */
+export async function create(): Promise<RLNLightInstance> {
```

Comment:
We don't need a function just doing `new something`

Reply:
this was following similar pattern to RLN
but since we are also not calling it `RLNLight`, removed

Reply:
I think we should be critical about present patterns and re-think them when we touch old code
Let's not add something that doesn't make much sense @waku-org/js-waku